### DISCUSSION TITLE
Docker Searching for MikroORM config file in build folder not dist folder

### DISCUSSION THREAD

**Discussion by Shivam-Pathak1** [mikro-orm/mikro-orm#1935] [Q&A]
I have the following project structure:
![image](https://user-images.githubusercontent.com/56876559/122191113-b90b3a80-ce92-11eb-838e-6d832732ffa0.png)

When Ido an npm build from my Dockerfile, the mikro-orm.postgres.config.ts should be translated into './dist/config/database/postgres/mikro-orm.postgres.config.js' . I can see the file being present there. But when I run my image, it gives the following error: 
Error: MikroORM config file not found in ['./src/config/database/postgres/mikro-orm.postgres.config.ts', './build/config/database/postgres/mikro-orm.postgres.config.ts', './mikro-orm.config.ts', './mikro-orm.config.js']

Ideally, the search for the file should be made on dist folder as I have configured my outdir as dist in tsconfig.json

tsconfig.json
```
{
  "compilerOptions": {
    "module": "commonjs",
    "declaration": true,
    "removeComments": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "incremental": true,
    "target": "es2017",
    "outDir": "./dist",
    "baseUrl": "./",
    "sourceMap": true,
    "strict": true,
    "strictPropertyInitialization": false,
    "noImplicitAny": false,
    "noLib": false,
    "esModuleInterop": true,
    "preserveConstEnums": true,
    "noUnusedLocals": false,
    "noUnusedParameters": false
  },
  "include": ["src/**/*"],
  "exclude": ["node_modules", "**/*.spec.ts"]
}

```

**Comment by B4nan** [comment]
The paths are controlled based on config in package.json, and it is totally up to you how this works - by default it only searches in root for the JS config.

**Reply by Shivam-Pathak1** [reply]
Oh thank you. Yes I could find the configPath in package json and corrected it.
I also noticed that I can configure migrations path in package json. Should migrations be inside src or in root folder? Migration scripts are being generated according to my entities so it's code first method. But for production docker I do not copy src folder to the container just the dist from development container. What do you advise?
